public/app/ui/pages/home/home.html

```diff
@@ -1,37 +1,41 @@
 <article id="home">
     <h1>Research Western Australian History</h1>
     <p>
-        This website provides access to the Royal Western Australian Historical Society's collection for research and public interest purposes. Searches can be performed on the <a href="/photographs/search">photographic</a> and <a href="/museum/search">museum</a> collections, as well as <a href="/memorials/search">memorial records</a> and the <a href="/library/search">library catalogue</a>.
+        Welcome to the gateway for searching the catalogues of The Royal Western Australian Historical Society’s historical collections.  The items embraced by these records have been collected and managed by the Society since 1926 and include a wealth of information on many aspects of Western Australian history dating back to European settlement in 1826, with some going back even further.  The catalogues cover the Library, Museum, Photographs and Public Memorials collections.  They have been made accessible via the website so that users can search them from anywhere in the world for historical research and general interest purposes.
```

Comment:
Given that this text is uneditable, I think it is too much.  Was it provided by the client?  Also the words "Library", "Museum" etc should be linked to the relevant search pages.